### PR TITLE
Change-log proposal for 4.5

### PR DESCRIPTION
**Pull request by thierry-martinez**
I think we are close to 4.5 release (and then 5.0 release for ppxlib). Here is a change-log proposal.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,32 @@
...
+* `Ppx_deriving_runtime.Option` compatibility module
+  #222
+  (Thierry Martinez, reviewed by Gabriel Scherer)
+
+4.4.1
+-----
+
+* Add support for OCaml 4.10
+  #211
+  (kit-ty-kate, reviewed by Gabriel Scherer)
```
Comment (gasche) [inline CHANGELOG.md:28]: @kit-ty-kate, should we rather use your github handle or a full name of your choice?

Comment (gasche) [inline CHANGELOG.md:28]: I have a slight preference for `review` instead of `reviewed`, but I'm fine with `reviewed` as well.

Comment (thierry-martinez) [reply CHANGELOG.md]: Fixed in https://github.com/ocaml-ppx/ppx_deriving/pull/228/commits/e860858be255ff5c8e0d510b4024b8e47dfbb00b

Comment (kit-ty-kate) [reply CHANGELOG.md]: full name is fine

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,32 @@
...
 =========
 
+4.5
+---
+
+* Add support for OCaml 4.11.
+  - `Ppx_deriving.string_of_{constant,expression}_opt` to destruct
+    `Pconst_string` in a version-independent way
+  #220, #222
+  (kit-ty-kate, Thierry Martinez, review by Gabriel Scherer)
```
Comment (kit-ty-kate) [inline CHANGELOG.md:11]:
```suggestion
  (Kate Deplaix, Thierry Martinez, review by Gabriel Scherer)
```

**Comment by thierry-martinez** [comment]
Thanks for reviewing! Merged.

**Pull request merged**